app/storage.js

```diff
@@ -0,0 +1,29 @@
+const { DefaultAzureCredential } = require('@azure/identity')
+const { BlobServiceClient } = require('@azure/storage-blob')
+const config = require('./config').storageConfig
+let blobServiceClient
+
+if (config.useConnectionStr) {
+  console.log('Using connection string for BlobServiceClient')
+  blobServiceClient = BlobServiceClient.fromConnectionString(config.connectionStr)
+} else {
+  console.log('Using DefaultAzureCredential for BlobServiceClient')
+  const uri = `https://${config.storageAccount}.blob.core.windows.net`
+  blobServiceClient = new BlobServiceClient(uri, new DefaultAzureCredential())
+}
+
+const container = blobServiceClient.getContainerClient(config.container)
+
+const getFileStream = async (filename) => {

```

Comment:
Would suggest that this function should initialise containers if no already initialised like similar services that interact with storage.  Otherwise when running this in isolation  or deploying to SND for the first time, container will need to be created manually.